README.md

```diff
@@ -30,6 +30,61 @@ Please raise a new pull request for new library additions and existing library u
 
 ## Adding a new or updating an existing library
 
+There is a simple way to add a library via single package.json only.
+
+You can follow these steps:
+
+1. Fork cdnjs repository to your GitHub account.
+
+2. Create a new folder named **libname** under `/ajax/libs`, then create a package.json under `/libname`.
+
+    > A tip: you can type `/libname/package.json` after cdnjs/ajax/libs/ to create them if you use web interface
```

Comment:
Don't quote this, why you quote your own words even in the first time it appears?